README.md

```diff
@@ -131,6 +139,116 @@ To start either of the interfaces in development mode:
   - Unit-tests and UI-tests for all interfaces; Run with `npm test` in each `interfaces/*`-folder.
   - Unit-tests and integration-tests for all services; Run with `npm test` in each `services/*`-folder.
 
+## Common problems with Local Environment set-up
+If the swagger UI is not accessible after installing docker and setting up services, we can take following steps to debug:
+1. `docker ps` to list running containers and their statuses
+2. `docker logs -f <container-name>` to check their statuses
+
+- If the errors are related to not being able to access/connect to 'database' then resetting the database by using `dropSchema: true` in `ormconfig.ts`.
+
+- In order to access docker container itself, execute command `sudo docker exec -it <container-name> sh` and verify installation of npm modules.
```

Comment:
```suggestion
- In order to access an individual docker container, execute command `docker-compose exec -it <container-name> sh` and verify installation of npm modules.
```

`sudo` shouldn't be necessary here. (Currently on our servers, it _is_; But that should be fixed there)